test/integration/scripting_spec.mjs

```diff
@@ -2425,4 +2425,49 @@ describe("Interaction", () => {
       );
     });
   });
+
+  describe("Value of event.change when a choice list is modified", () => {
+    let pages;
+    let otherPages;
+
+    beforeAll(async () => {
+      otherPages = await Promise.all(
+        global.integrationSessions.map(async session =>
+          session.browser.newPage()
+        )
+      );
+      pages = await loadAndWait("issue17998.pdf", getSelector("7R"));
+    });
+
+    afterAll(async () => {
+      await closePages(pages);
+      await Promise.all(otherPages.map(page => page.close()));
+    });
+
+    it("must check the properties of the event", async () => {
+      await Promise.all(
+        pages.map(async ([browserName, page], i) => {
+          await page.waitForFunction(
+            "window.PDFViewerApplication.scriptingReady === true"
+          );
+
+          for (const [value, expected] of [
+            ["b", "change=B,changeEx=b,value=A"],
+            ["c", "change=C,changeEx=c,value=B"],
+            ["a", "change=A,changeEx=a,value=C"],
+          ]) {
+            await page.select(getSelector("7R"), value);
+            await page.waitForFunction(
```

Comment:
Similar to what we did in https://github.com/mozilla/pdf.js/pull/17974/files#diff-c0973b2c3b147db1eccd21c6b1af307f33a7e5d2658b4a0738a2386470b9c0caL2271, could we change this to the template string approach since it overall seems easier to read?

```js
await page.waitForFunction(`${getQuerySelector("10R")}.value !== ""`);
```